dex/models/trades/sei/platforms/jelly_swap_sei_base_trades.sql

```diff
@@ -0,0 +1,22 @@
+{{
+    config(
+        schema = 'jelly_swap_v2_sei',
+        alias = 'base_trades',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['tx_hash', 'evt_index'],
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
+    )
+}}
+
+{{
+    balancer_compatible_v2_trades(
```

Comment:
Do you think we may simplify this macro?

Reply:
only low hanging fruit I see is discarding this parameter:
`Vault_evt_Swap = 'Vault_evt_Swap'`
Open to any other suggestions you may have, though

Reply:
lets save any potential refactor of a macro to separate PR. i'm also comfortable with vini having final say on the balancer macros, as our lead on that 🙏